lib/galaxy/datatypes/data.py

```diff
@@ -988,7 +988,7 @@ def get_file_peek( file_name, is_multi_byte=False, WIDTH=256, LINE_COUNT=5, skip
         if not line_wrap:
             if '\n' in line:
                 i = line.index( '\n' )
-                last_line = line[i:]
+                last_line = line[i + 1:]
```

Comment:
Since `line` is read with [readline()](https://docs.python.org/2/library/stdtypes.html#file.readline), `'\n'` should always be its last character (if it is present). So `last_line` should be always empty and it can be removed from everywhere in this function.

@blankenberg Am I missing something?